### PR TITLE
Add save/restore scroll pos for themes list.

### PR DESCRIPTION
**Pull request by Zangetsu38**
- Add save and restore scroll pos for can resore it.

Very needed if you have very lot diferent theme in list for it not re go on 0 after set back.

### CODE REVIEW

File: vita3k/gui/src/themes.cpp [hunk 1]
```diff
@@ -419,6 +419,8 @@ void draw_start_screen(GuiState &gui, HostState &host) {
 
 static std::string popup, menu, selected, title, start, delete_user_background;
 static ImGuiTextFilter search_bar;
+static float scroll_pos;
+static bool set_scroll_pos;
```
Comment (1whatleytay) [inline vita3k/gui/src/themes.cpp:423]: sad static

File: vita3k/gui/src/themes.cpp [hunk 2]
```diff
@@ -490,8 +496,16 @@ void draw_themes_selection(GuiState &gui, HostState &host) {
...
     } else if (menu == "theme") {
+        // Theme List
         if (selected.empty()) {
             title = "Theme";
+
+            // Set Scroll Pos
+            if (set_scroll_pos) {
+                ImGui::SetScrollY(scroll_pos);
+                set_scroll_pos = false;
+            }
```
Comment (1whatleytay) [inline vita3k/gui/src/themes.cpp:507]: Strange to me why you cant just do `ImGui::SetScrollY(scroll_pos);` on line 790.

Comment (Zangetsu38) [reply vita3k/gui/src/themes.cpp:507]: is outside begin_child, you can't, if you set this here, you change position scroll inside begin :D

File: vita3k/gui/src/themes.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/themes.cpp:423]: static is perfect on general :)

**Pull request merged**